### DISCUSSION TITLE
Bold and italic issues

### DISCUSSION THREAD

**Discussion by sarmong** [nvim-neorg/neorg#305] [Q&A] [answered]
Not sure if this is expected, but some of the modifiers on my system look like this.

![image](https://user-images.githubusercontent.com/42828375/148698500-3f567f45-dc8c-435a-9c54-0464134cdf22.png)

The first three, for example, are quite difficult to distinguish. 

Can I make them more usable? I am not sure where to look for - colorsheme, treesitter issues, fonts, etc.

**Comment by max397574** [comment, marked as answer]
This should be a colorscheme issue as far as I can tell.
Can you try `:highlight NeorgMarkupItalic/Bold/Underline` ?
if they all look the same (like the top three in the screenshot) then it's a colorscheme issue

**Reply by sarmong** [reply]
```
NeorgMarkupBold xxx links to TSStrong
NeorgMarkupItalic xxx links to TSEmphasis
NeorgMarkupUnderline xxx links to TSUnderline
```

This seems correct, right?

I installed some other colorschemes and they didn't work. But now I see why. If I change a colorscheme right inside neovim from the one that doesn't have this things set up to the one that does, bold and underline still don't work.
But if I set the right colorscheme in the config and reload vim, it shows bold and other things correctly.

So, I need a colorscheme with configured TSStrong, TSEmphasis and TSUnderline, right?

**Reply by sarmong** [reply]
Or rather, how do I set it up myself? I didn't find any info in wiki.

**Reply by max397574** [reply]
This is correct
But is TSStrong/Emphasis/Underline bold,italic,underlined for you?
If not I think you could do it with
`hi TSStrong gui=bold`
`hi TSEmphasis gui=italic`
`hi TSUnderline gui=underline`
